hangman.js

```diff
@@ -0,0 +1,132 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Overall very impressive work! I appreciate the fact that you went extra mile with theme, aesthetics, and coding. Only feedback is to use more functions for future projects to make code less DRY and also better organized. Great work and looking forward to see more in the future!